package.json

```diff
@@ -22,18 +22,19 @@
   },
   "homepage": "https://github.com/not-an-aardvark/eslint-plugin-prettier#readme",
   "dependencies": {
-    "prettier": "^0.11.0",
     "requireindex": "~1.1.0"
   },
   "peerDependencies": {
-    "eslint": "^3.14.1"
+    "eslint": "^3.14.1",
+    "prettier": ">= 0.11.0"
   },
   "devDependencies": {
     "eslint": "^3.14.1",
     "eslint-config-not-an-aardvark": "^2.0.0",
     "eslint-plugin-eslint-plugin": "^0.2.1",
     "eslint-plugin-node": "^3.0.5",
-    "mocha": "^3.1.2"
+    "mocha": "^3.1.2",
+    "prettier": ">= 0.11.0"
```

Comment:
The version range does not _technically_ have to be `>= 0.11.0` here; `^0.11.0` could work as well. I thought it might be a good idea to you the same range as the peerDependency, though, to make sure that the tests pass when using the latest version allowed by the peerDependency. When I made the PR, the latest version of prettier on npm was 0.11.0. Now, when the PR was merged, 0.13.1 has been released, with slightly different output. I didn’t realize that this repo uses prettier for linting itself, which caused Travis to fail on the merge commit. Thoughts?

- Change the devDependency version constraint?
- Remove prettier from the eslint rules of this module?
- Just live with it, and re-format the files every now and then?